### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by DeathlyBower959** [bozbez/win-capture-audio#39]
I suggest adding to the `Capture all audio EXCEPT the selected window` feature, and allow it to select multiple windows to exclude.

**Comment by bozbez** [comment]
Yeah indeed, but this issue here is that the exclude mode comes directly from the API so the only way to implement this is to change the feature so we enumerate and capture all audio sessions that don't match the window selection.

Will consider for v2.1.

**Comment by DeathlyBower959** [comment]
All righty cool

**Comment by bozbez** [comment]
Now actually possible with the mixer stuff in [v2.2.0-beta](https://github.com/bozbez/win-capture-audio/releases/tag/v2.2.0-beta), needs some thought about how to implement in the UI.

**Comment by DeathlyBower959** [comment]
For the UI, if possible I would make the dropdown multi select. Otherwise, I would just have a dropdown, with a plus sign to add more exclusions. Clicking the plus sign would add another dropdown to select more sources to exclude.
